woocommerce/woocommerce-gutenberg-products-block
# Remove the ToggleButtonControl in favor of ToggleGroupControl

This PR replaces the `ToggleButtonControl` Product Blocks component in favor of the core `ToggleGroupControl`.


Fixes #5873 

### Screenshots
|Before|After|
|-|-|
|![OLD](https://user-images.githubusercontent.com/905781/156161666-57c4baed-9d17-4c65-8fc8-4a8d2316dfde.jpg)|![new](https://user-images.githubusercontent.com/905781/156161720-b962056e-be7f-40ca-9173-52a72443b01a.jpg)|

### Manual Testing
How to test the changes in this Pull Request:

1. Check out this branch and compile the changes
2. Activate a **block theme**, like Twenty Twenty Two
3. Create a new page, and add the following blocks:
- Active Filters
- Attribute Filter
- Price Filter
- Product Categories
- Reviews
4. Ensure that options are using the new component and are working correctly

### User Facing Testing
These are steps for user testing (where "user" is someone interacting with this change that is not editing any code).


#### Automated Tests
